src/main/java/net/minestom/server/event/player/PlayerBlockPlaceEvent.java

```diff
@@ -149,6 +152,16 @@ public BlockPosition getBlockPosition() {
         return hand;
     }
 
+    /**
+     * Original block on the {@link BlockPosition} which the player is trying to replace.
+     * Mostly {@link Block#AIR}, but can be different.
+     *
+     * @return an ID of block on the location
+     */
+    public short getOriginalBlockStateId() {
```

Comment:
I think this should be `getOldBlockStateId`, for the same reason as in `PlayerMoveEvent`

Reply:
Agree, I will fix that. :)